common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -279,30 +299,11 @@ export interface IQueueMessage {
 /**
  * Interface for nack content.
  */
-export interface INackContent {
-    /**
-     * An error code number that represents the error. It will be a valid HTTP error code.
-     * 403 errors are non retryable and client should acquire a new identity before reconnection.
-     * 400 errors are always immediately retriable
-     * 429 errors are retriable or non retriable (depends on type field).
-     */
-    code: number;
-
+export interface INackContent extends IServerError {
```

Comment:
i wouldn't extend any interface here, as it is hard to see what's breaking and what's not. this adds require fields, but we can't use partial either as message need to remain require